### ISSUE TITLE
Add rcannood as contributor?

### ISSUE THREAD

**Issue by lazappi** [theislab/zellkonverter#88]
Hey @rcannood 

I was just merging another PR and something made me realise I don't think I asked whether you want to be added as a contributor after #76. If you would like to be added feel free to open a PR (or send me the details and I can do it). If not I'll just close this.

Cheers

**Comment by rcannood** [comment]
I' ve created a PR. Thanks @lazappi !
